x-pack/solutions/chat/plugins/workchat-app/server/services/orchestration/agent_graph.ts

```diff
@@ -48,14 +47,9 @@ export const createAgentGraph = async ({
 
   const toolsProvider = new ToolsProvider({ session, logger });
 
-  const searchTool = createSearchAgentTool({
-    toolsProvider,
-    chatModel,
-    logger,
-  });
-  const builtInTools = await toolsProvider.getBuiltInTools();
+  const allTools = await toolsProvider.getAllTools();
```

Comment:
This removes usage of the research assistant _for now_. We can add this back when we have more cycles to make it a little more performant and reliable.